src/app/shared/classes/releases.ts

```diff
@@ -51,17 +51,19 @@ export const releases: ReleaseItem[] = [{
 }, {
   version: 'EnergyPlus 22.2.0',
   url: 'https://github.com/NREL/EnergyPlus/releases/tag/v22.2.0'
+}, {
+  version: 'EnergyPlus 23.1.0',
+  url: 'https://github.com/NREL/EnergyPlus/releases/tag/v23.1.0'  
 }];
 
 export const latestRelease: CurrentRelease = {
-  version: '23.1.0',
-  date: 'March 31, 2023',
-  mac12: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Darwin-macOS12.1-x86_64.dmg',
-  mac11: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Darwin-macOS11.6-x86_64.dmg',
-  mac10: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Darwin-macOS10.15-x86_64.dmg',
-  windows_main: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Windows-x86_64.exe',
-  windows_32: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Windows-i386.exe',
-  ubuntu_22: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Linux-Ubuntu22.04-x86_64.run',
-  ubuntu_20: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Linux-Ubuntu20.04-x86_64.run',
-  ubuntu_18: 'https://github.com/NREL/EnergyPlus/releases/download/v23.1.0/EnergyPlus-23.1.0-87ed9199d4-Linux-Ubuntu18.04-x86_64.run'
+  version: '23.2.0',
```

Comment:
Updated current release to 23.2.  One note here: since we standardize on a specific file name pattern, is it possible to simply define the version number string and SHA as a string and use them in these strings so we don't have to update a whole list here?  I don't have this building locally or I would try it myself.  And it's not important, so feel free to ignore... :)